src/components/field-label/field-label.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import invariant from 'tiny-invariant';
+import IconButton from '../buttons/icon-button';
+import { InformationIcon } from '../icons';
+import Text from '../typography/text';
+import Label from '../label';
+import Constraints from '../constraints';
+import Spacings from '../spacings';
+import styles from './field-label.mod.css';
+
+type HorizontalConstraintOfFieldLabel = 's' | 'm' | 'l' | 'xl' | 'scale';
+type FieldLabelProps = {
+  title: React.ReactNode;
```

Comment:
I initially thought to go with `React.ReactNode | string` but looking up the types helps a lot.

<img width="536" alt="cleanshot 2019-01-23 at 11 40 14 2x" src="https://user-images.githubusercontent.com/1877073/51601596-1e96af00-1f05-11e9-92df-b0d9b196387c.png">

This means that a string and it being optional seems to be part of a `ReactNode`.